test/views/editLiability/disposeLiabilityBankDetailsSpec.scala

```diff
@@ -63,7 +63,6 @@ implicit val authContext: StandardAuthRetrievals = organisationStandardRetrieval
       Then("The fields should have the correct titles")
       And("No data is populated")
 
-      assert(document.getElementById("hasUKBankAccount-id").text() === "Is the bank account in the UK? Yes No")
```

Comment:
Can you change the test rather than remove it. We need to test that something is there.